Cargo.toml

```diff
@@ -65,6 +65,7 @@ members = [
 structopt = "0.2.10"
 failure = "0.1.3"
```

Comment:
Can we get rid of the depenency here as well?

Reply:
Unfortunately, two In-depth chapters are still using `failure`. I managed to change it to using `anyhow` in `signals-channels.rs`. but I couldn't change it in `machine-communication-convey.rs` which uses the `convey` crate. The error message is below.

```
error[E0277]: the trait bound `convey::Error: std::error::Error` is not satisfied
  --> src/in-depth/machine-communication-convey.rs:20:48
   |
20 |         convey::new().add_target(json::stdout()?)
   |                                                ^ the trait `std::error::Error` is not implemented for `convey::Error`
   |
   = note: required because of the requirements on the impl of `std::convert::From<convey::Error>` for `anyhow::Error`
   = note: required by `std::convert::From::from`
```

Reply:
Ah, right. Okay, let's fix this in a future PR then!